samples/multiplatform/kmp-ios-app/kmp-ios-app/SceneDelegate.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  SceneDelegate.swift
+//  kmp-ios-app
+//
+//  Created by Taso Dane on 4/5/20.
+//
+
+import UIKit
+import SwiftUI
+import kmp_lib_sample
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+
+    var window: UIWindow?
+
+
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
+        // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
+        // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
+        let query = GithubRepositoriesQuery(
+            repositoriesCount: 50,
+            orderBy: RepositoryOrderField.updatedAt,
+            orderDirection: OrderDirection.desc
+        )
+        let requestPayload = "{" +
+        "\"operationName\": \"" + query.name().name() + "\", " +
+        "\"query\": \"" + query.queryDocument() + "\", " +
+        "\"variables\": " + query.variables().marshal() +
+        "}"
```

Comment:
What about adding `fun Query.toJson(): String` to factor this code ?

Reply:
Yes! definitely. I got this from our docs. I compared it with what we have in runtime. Runtime also adds more in case autoPersist is enabled. We should do that and move this to common code

Reply:
Created a follow-up on this https://github.com/apollographql/apollo-android/issues/2137